cg/meta/demultiplex/utils.py

```diff
@@ -246,13 +256,57 @@ def is_syncing_complete(source_directory: Path, target_directory: Path) -> bool:
     return are_all_files_synced(files_at_source=files_at_source, target_directory=target_directory)
 
 
+def confirm_illumina_flow_cell_sync(source_directory: Path, target_directory: Path) -> None:
+    """Checks if all relevant files from an Illumina sequencing run have been transferred."""
+    for source_flow_cell in Path(source_directory).iterdir():
+        target_flow_cell = Path(target_directory, source_flow_cell.name)
+        if is_flow_cell_sync_confirmed(target_flow_cell):
+            LOG.debug(f"Flow cell {source_flow_cell} has already been confirmed, skipping.")
+            continue
+        if is_syncing_complete(
+            source_directory=source_flow_cell,
+            target_directory=Path(target_directory, source_flow_cell.name),
+        ):
+            create_copy_complete_file(target_flow_cell)
+
+
+def confirm_nanopore_flow_cell_sync(source_directory: Path, target_directory: Path) -> None:
+    """Checks if all relevant files from a Nanopore sequencing run have been transferred."""
+    for source_flow_cell in get_nanopore_flow_cell_directories(source_directory):
+        target_flow_cell = Path(
+            target_directory, get_nanopore_flow_cell_relative_path(source_flow_cell)
+        )
+        if is_flow_cell_sync_confirmed(target_flow_cell):
+            LOG.debug(f"Flow cell {source_flow_cell} has already been confirmed, skipping.")
+            continue
+        if is_syncing_complete(
+            source_directory=source_flow_cell,
+            target_directory=target_flow_cell,
+        ):
+            create_copy_complete_file(target_flow_cell)
+            create_nanopore_trigger_file(
+                run_name=target_flow_cell.name,
+                trigger_directory=Path(
+                    target_directory, NanoporeDirsAndFiles.SYSTEMD_TRIGGER_DIRECTORY
+                ),
+            )
+
+
 def get_flow_cell_id(flow_cell_dir_name: str) -> str:
     """Return the flow cell id from the flow cell directory name.
     Example: 230802_A00689_0857_BHGTMFDSX7 -> HGTMFDSX7
     """
     return flow_cell_dir_name.split("_")[-1][1:]
 
 
+def get_nanopore_flow_cell_directories(nanopore_data_directory: Path) -> list[Path]:
+    return list(Path(nanopore_data_directory).glob("*/*/*"))
```

Comment:
```suggestion
    return list(Path(nanopore_data_directory).glob(THREE_LEVELS_DOWN))
```

This constant was created somewhere down.

Reply:
True moved and used it 👍